src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -762,16 +764,44 @@ private async Task ChangeOrganizationSponsorship(Organization org, OrganizationS
         var daysUntilDue = sub.DaysUntilDue;
         var chargeNow = collectionMethod == "charge_automatically";
         var updatedItemOptions = subscriptionUpdate.UpgradeItemsOptions(sub);
+        var isPm5864DollarThresholdEnabled = _featureService.IsEnabled(FeatureFlagKeys.PM5864DollarThreshold);
 
         var subUpdateOptions = new Stripe.SubscriptionUpdateOptions
         {
             Items = updatedItemOptions,
-            ProrationBehavior = "always_invoice",
+            ProrationBehavior = isPm5864DollarThresholdEnabled && invoiceNow
```

Comment:
⚠️ This doesn't quite look right. We want to `always_invoice` if the feature flag isn't true _or_ if `invoiceNow` is true.

Reply:
resolved